content/source/docs/enterprise/vcs/bitbucket-server.html.md

```diff
@@ -30,6 +30,11 @@ TFE uses webhooks to get new configurations. To support this, Bitbucket Server n
 
 Leave the page open in a browser tab, and remain logged in as an admin user.
 
+### Important Notes: 
+
+- The Plugin is disabled by default. Clicking `Enabled` will toggle the plugin on. 
```

Comment:
I'd suggest moving this up to item 3 in the list above, since that's where we first mention enabling it.

Reply:
👍